src/tag/tag.scss

```diff
@@ -75,6 +77,7 @@
 
   &.iui-truncate {
     @include iui-line-clamp(1);
+    padding: 4px; // Needed to show full focus box
```

Comment:
I've made some changes and was able to remove the padding. Only potential downside is the outline is right up against the text, it might be a little hard to read.
![Screen Shot 2022-01-03 at 2 49 11 PM](https://user-images.githubusercontent.com/849817/147973507-dd1a183a-6c14-4947-951d-99523ee9928f.png)

Reply:
Why not add the rule to the tag instead of the container?

Reply:
I was able to add a 2px margin to all `.iui-basic` tags nested within the tag container & a 1px offset on focus outline, now the outline isn't touching the text.
![Screen Shot 2022-01-05 at 11 39 23 AM](https://user-images.githubusercontent.com/849817/148254723-0e78ba86-e0f2-46e3-86d4-7831d832f570.png)

Reply:
Actually, I was able to change it to only target `<a class="iui-tag iui-basic">` since the non-`<a>` will not have a focus.  This reduces visual changes in the tile components which use the non-`<a>` tags.
![Screen Shot 2022-01-05 at 11 46 42 AM](https://user-images.githubusercontent.com/849817/148255791-da8f7ede-35c7-4537-9b9c-713a1a4cf2f4.png)

Reply:
That still doesn't address my concern:
> Why not add the rule to the tag instead of the container?

I'm saying the margin/focus rules don't need to be nested in iui-tag-container.

Reply:
I tried moving the margin / focus stuff outside of `.iui-tag-container` but ran into some issues.  I was able to solve it, but with a bit more SCSS than was in there originally, so I reverted it back.

Reply:
I pushed a fix with even fewer lines of code. Please verify it and feel free to merge this PR